src/identities/identities.controller.ts

```diff
@@ -72,6 +76,28 @@ export class IdentitiesController {
     logger.setContext(IdentitiesController.name);
   }
 
+  @Post('register')
+  @ApiOperation({
+    summary: 'Register Identity',
+    description:
+      'This endpoint allows registering a new Identity. The transaction signer must be a CDD provider. This might create an Authorization Request which has to be accepted by the target Account.',
```

Comment:
```suggestion
      'This endpoint allows registering a new Identity. The transaction signer must be a CDD provider. This will create Authorization Requests which have to be accepted by any secondary accounts if they were specified.',
```

(nit) I think its more accurate to say this will create authorization requests for secondary accounts which will need to be accepted.

I think the primary account doesn't get an auth, it just has the claim made for it.